File: sum_all_3_and_5_to_1000.py
#sum of all numbers divisble by 3 and 5 till 1000
def sum_all(num):
    sum_num = 0
    
    for i in range(0,num+1):
        if i % 3 == 0 and i % 5 == 0:
            sum_num = sum_num + i

    return sum_num

print sum_all(1000)

File: sumAll3And5to1000.js
/* sum of all numbers divisble by 3 and 5 till 1000*/
// now solve in Python

function sumAll(num){
	var sumNums = 0;

	for(var i = 0; i < num; i++){
		if(i % 3 === 0 && i % 5 === 0){
			sumNums = sumNums + i;
		}
	}

	return sumNums;
}

var requiredNum = 1000;

console.log(sumAll(requiredNum));